libraries/core/lib/search/objective/ObjectiveFunction.ts

```diff
@@ -24,18 +24,18 @@ import { SearchSubject } from "../SearchSubject";
  *
  * @author Mitchell Olsthoorn
  */
-export interface ObjectiveFunction<T extends Encoding> {
+export abstract class ObjectiveFunction<T extends Encoding> {
```

Comment:
This class can be converted back in an interface

Reply:
Fixed.